cernopendata/modules/fixtures/data/docs/delphi-guide-docker/delphi-guide-docker.md

```diff
@@ -1,80 +1,72 @@
 ## Running DELPHI software from a container
 An alternative way to run DELPHI software is to use a container image.
-The following guide assumes that you are using [Podman](https://podman.io/), nevertheless it is expected to work in a similar way with other container management technologies, such as Docker.
-The images are automatically created from a CI when the software stack is being build, and contain the full software stack.
+The following guide assumes that you are using [Podman](https://podman.io/), nevertheless it is expected to work in a similar way with other container management technologies, such as [Docker](https://www.docker.com/).
+The required images to run the DELPHI software are automatically created from a CI when the software stack is being build, and contain the full software stack.
 
 ## Available images
-Images are available in the registry of the deployment project at CERNs gitlab instance, https://gitlab.cern.ch/delphi/deployment/container_registry
+Images are available in the registry of the `DELPHI deployment` project at [CERNs gitlab instance](https://gitlab.cern.ch/delphi/deployment/container_registry).
 
-In this guide we will use the latest Alma9 image.
+In this guide we will use the latest `Alma9` image.
 
 ## Starting the container
-In this guide, we will assume to be running on a Linux desktop, which has podman and podman-docker installed. Some of the applications require a graphical interface, namely the CERNLIB applications paw, paw++ and kxterm, as well as the event display.
+In this guide, we will assume to be running on a Linux desktop, which has the `podman` and `podman-docker` packages installed. Some of the applications require a graphical interface,
+namely the CERNLIB applications `paw`, `paw++` and `kxterm`, as well as the [DELPHI event display](delphi-guide-delgra).
 
-If you want to run on any data samples, please download them first. You can then attach them using the -v option of the `podman run` command. Please check the podman documentation for more details.
+If you want to run on any data samples, please download them first. You can then attach them using the -v option of the `podman run` command. Please check the podman documentation for more details. The `Alma9` image we use in this guide can also access the data directly from the `EOS` file system.
```

Comment:
EOS is not something people type, so I would remove backquotes here and elsewhere.